dart/collision/ode/detail/OdeMesh.cpp

```diff
@@ -58,7 +58,7 @@ OdeMesh::OdeMesh(
       mOdeTriMeshDataId,
       mVertices.data(),
       3*sizeof(double),
-      static_cast<int>(mVertices.size()),
+      static_cast<int>(mVertices.size()/3),
```

Comment:
Could you elaborate a little why this is needed?

Reply:
Sure. The signature of that function is
```c++
ODE_API void dGeomTriMeshDataBuildDouble1(dTriMeshDataID g, 
                                  const void* Vertices,  int VertexStride, int VertexCount, 
                                  const void* Indices, int IndexCount, int TriStride,
                                  const void* Normals);
```
The fourth argument is the number of vertices. `mVertices` actually has 3 elements (the x, y, z components) per vertex so we have to divide its size by 3 to get the number of vertices. I checked against [Gazebo's implementation](https://bitbucket.org/osrf/gazebo/src/default/gazebo/physics/ode/ODEMesh.cc#lines-141) to make sure.